engine/runtime-compiler/src/main/scala/org/enso/compiler/data/BindingsMap.scala

```diff
@@ -734,13 +734,18 @@ object BindingsMap {
     override val name: String,
     params: Seq[String],
     members: Seq[Cons],
-    builtinType: Boolean
+    builtinType: Boolean,
+    isPrivate: Boolean
```

Comment:
Docs for the new `isPrivate` parameter: "A type is considered private, if all its constructors are private, or if it is defined in a private module. Constructors and fields of a private type will not be accessible outside the same project".

Better have the docs duplicated than none at all.